src/ebms/message.js

```diff
@@ -14,6 +15,8 @@ class Message {
     );
   }
 
+  pieceJointePresente = () => this.pieceJointe.constructor.name === 'PieceJointe';
```

Comment:
Pour une autre PR peut-être : plutôt faire un appel du style `this.pieceJointe.presente()`